MAUI/ImageEditor/imagefilter.md

```diff
@@ -24,7 +24,7 @@ Using the image editor control, apply various effects such as Brightness, Hue, S
 The EffectValue are the corresponding ImageEffect values, which varies for each effect, and they are explained as follows.
 
 N> The Image Effect enum also contains “None” option, which removes all the previously applied effects, which are not saved.

```

Comment:
The `ImageEffect`(link) enum also contains `ImageEffect.None`(link) option, which removes all the previously applied effects, which are not saved.

Reply:
added